pyproject.toml

```
@@ -63,8 +63,8 @@ lib = []
 model = []
 extra-lint-examples = [
   "pylint >= 2.17.3, < 4",
-  "pytest >= 7.3.0, < 8",
-  "sybil >= 5.0.3, < 6",
+  "pytest >= 7.4.2, < 9",
+  "sybil >= 6.0.3, < 7",
```

Ideally we should only widen these to avoid making this a breaking change and still support pytest 7.x (if we are not affected by the breaking changes in 8):

```suggestion
  "pytest >= 7.3.0, < 9",
  "sybil >= 5.0.3, < 7",
```

I did some local testing with this and it seems to work.